ruby-challenges.rb

```diff
@@ -19,8 +32,18 @@
 nums_array2 = [25, 17, 47, 11]
 # Expected output: 100
 
+def sum(numbers)
+  total = 0
+  i = 0
+  while i < numbers.count
+    total += numbers[i]
+    i += 1
+  end
+  total
+end
 
-
+p sum(nums_array1)
+p sum(nums_array2)
```

Comment:
Nice!  Ruby also has a built-in method called .sum that will do this heavy lifting for you.  When in doubt, check if there is a built-in Ruby method.  Everything that took a ton of work in JS becomes so simple in Ruby.